package.json

```diff
@@ -24,9 +24,9 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "@ember/render-modifiers": "^1.0.2",
+    "@ember/render-modifiers": "^2.0.0",
```

Comment:
I think we might want something like this

https://github.com/adopted-ember-addons/ember-sortable/pull/449/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R27

Reply:
Yes I saw that too, not sure why though, is it to keep compatibility with Node 10? Anyway, I have updated this branch to allow both majors too.